themes/frontierline-nightly/content-views/content.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Display a full post, no featured image.
```

Comment:
There is now option to hide the featured image in the parent theme. The only difference from the parent is the `#fx-footer` at the bottom.